sunpy/map/tests/test_compositemap.py

```diff
@@ -34,6 +35,9 @@ def test_type_of_arguments_composite_map(composite_test_map):
         sunpy.map.CompositeMap(23, composite=True)
     assert str(excinfo.value) == 'CompositeMap expects pre-constructed map objects.'
 
+def test_autoalign_composite_map(composite_test_map):
+    assert isinstance(composite_test_map.plot()[0], mpl.image.AxesImage)
+    assert isinstance(composite_test_map.plot()[1], mpl.collections.QuadMesh)
```

Comment:
since `compositemap.plot` use the axes projected by first map
so for `composite_test_map.plot()[0]` will be instance of `mpl.image.AxesImage as ax.wcs==map[0].wcs,`
-> `autoalign = False`
`composite_test_map.plot()[1]` will be instance of `mpl.collections.QuadMesh as ax.wcs != map[0].wcs,`
-> `autoalign = True`